### PR TITLE
Support Vimeo with OAuth2

### PR DESCRIPTION
**Pull request by amarchen**
I only verified an app-specific token. Works well though.

### CODE REVIEW

**Comment by kullfar** [comment]
I will need test user and app credentials to run the example before merging to the master.
https://github.com/scribejava/scribejava/wiki/FAQ#when-will-you-support-insert_api_here-

File: src/main/java/org/scribe/builder/api/VimeoApi20.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package org.scribe.builder.api;
```
Comment (kullfar) [inline src/main/java/org/scribe/builder/api/VimeoApi20.java]: looks like very-very old base branch. rebase, pleas

**Comment by amarchen** [comment]
You can test with these temporary credentials:
```
    String apiKey = "e451c9567581e50e40431cf9dd2ba121e0afe39b";
    String apiSecret = "Rhj6Vx9HxABzMNSbiO3NwXZyff5JFijQvg+0F/4SPC6XSGELcRTz972Ju7Z+/FEpm1XSHhbL58a7MuIczhabWr5ldCBsRgLjJ2jkUiCM3I8OkEmwIrIj1SQfbYfXwHam";
    String accessToken = "0a712c420731944980575df421f8bcd8";
```

**Comment by kullfar** [comment]
outdated

**Pull request closed without merging**